internal/provisioner/plain/main.go

```diff
@@ -64,7 +69,7 @@ func main() {
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.StringVar(&systemNamespace, "system-namespace", "rukpak-system", "Configures the namespace that gets used to deploy system resources.")
-	flag.StringVar(&unpackImage, "unpack-image", "quay.io/operator-framework/plain-provisioner:latest", "Configures the container image that gets used to unpack Bundle contents.")
+	flag.StringVar(&unpackImage, "unpack-image", defaultProvisionerImage+":"+defaultProvisionerImageTag, "Configures the container image that gets used to unpack Bundle contents.")
```

Comment:
Also, as an aside, I'm not sure how much value the variables `defaultProvisionerImage` and `defaultProvisionerImageTag` are providing here since they are only called once and don't seem resuable. My preference would be not to introduce extra variables and simply update the embedded image name to point to `local` instead.